llm_bench/python/requirements_2024.3.txt

```diff
@@ -0,0 +1,21 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+numpy==1.26.4
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+auto_gptq==0.7.1
+pillow==10.4.0
+torch==2.4.0+cpu
+torchvision==0.19.0+cpu
+transformers==4.42.4
+diffusers==0.29.2
+openvino==2024.3.0
+#optimum is in dependency list of optimum-intel
+git+https://github.com/huggingface/optimum-intel.git@439d61f79cf55d5d0b28334f577b6ac3c5ced28f#egg=optimum-intel

```

Comment:
this is not final validation commit of optimum-intel for release (with GPU accuracy bugs), please use pinned in branch commit

Reply:
@eaidova Do you mean this one? https://github.com/openvinotoolkit/openvino.genai/blob/releases/2024/3/llm_bench/python/requirements.txt#L14

Reply:
Updated